libdevcore/CommonIO.h

```diff
@@ -57,4 +58,6 @@ std::string toString(_T const& _t)
 	return o.str();
 }
 
+boost::filesystem::path weaklyCanonical(boost::filesystem::path const& _path);
```

Comment:
Should this be more reflective that it is `boost::filesystem` related function?

Reply:
I mean should the name of this function contain `filsystem`, `boost` or both?

Reply:
What about a comment, stating that it's a partial re-implementation of boost::weakly_canonical and to be replaced with it, once we have boost>=1.60?

Reply:
I am just not fond that it is a function name without context. It doesn't mean anything when looking at it in the code.

Reply:
What about ``weaklyCanonicalFilesystemPath``?

Reply:
Sounds good!